elastic/beats
# [Heartbeat] Allow browser monitors on ARM

This patch removes the now unnecessary restriction that the extra syscall perms only apply to x86. This was necessary IIRC on centos, where IIRC attempting these syscalls on ARM would cause issues. 

It seems odd that would be the case, regardless, this is tested and works on a graviton2 processor.

Resolves https://github.com/elastic/beats/issues/29034

## Checklist


## How to test this PR locally

Attempt to run synthetic monitors with an ARM processor. I usually use this one: https://github.com/elastic/synthetics-demo/blob/main/heartbeat/monitors.d/browser-inline.yml . I'd also check that things look good in kibana. This will require synthetics 1.0.0-beta.24 or higher
